src/app/users/views/summaryPartial.ejs

```diff
@@ -1,128 +1,129 @@
+                <dt class="label">Account status:</dt>
+                <dd><%= user.status.description %></dd>
+                <dt class="label">Number of logins in last 12 months:</dt>
+                <dd><%= user.loginsInPast12Months.successful %></dd>
+            </dl>
+        </div>
+        <div class="col-4">
+            <% if(!locals.isInvitation) { %>
+                <aside>
+                    <h2 class="heading-medium">Actions</h2>
+                    <ul class="list">
+                        <li><a href="associate-organisation">Add organisation</a></li>
+                        <% if (locals.organisations.length > 0) { %>
+                            <li><a href="select-organisation">Add services</a></li>
+                        <% } %>
+                        <li><a href="edit-email">Change user's email address</a></li>
+                        <% if(locals.user.status.id === 0) { %>
+                            <li><a href="confirm-reactivation">Reactivate user</a></li>
+                        <% } else { %>
+                            <li><a href="confirm-deactivation">Deactivate user</a></li>
+                        <% } %>
+                        <li><a href="edit-profile">Edit user's details</a></li>
+                        <li><a href="web-service-sync">Send WS Sync</a></li>
+                    </ul>
+                </aside>
+            <% } else if (locals.isInvitation && !locals.user.deactivated) { %>
+                <aside>
+                    <h2 class="heading-medium">Actions</h2>
+                    <ul class="list">
+                        <li><a href="associate-organisation">Add organisation</a></li>
+                        <% if (locals.organisations.length > 0) { %>
+                            <li><a href="select-organisation">Add services</a></li>
+                        <% } %>
+                        <li><a href="edit-email">Change user's email address</a></li>
+                        <li><a href="confirm-invitation-deactivation">Deactivate user</a></li>
+                        <li><a href="resend-invitation"> Resend invitation email</a></li>
+                    </ul>
+                </aside>
+            <% } %>
+        </div>
     </div>
-</div>
-
-<nav>
-    <ul class="tabs" role="tablist">
-
-        <% if(locals.area === 'organisations') { %>
-        <li class="active">Organisations</li>
-        <% } else { %>
-        <li><a href="organisations" role="tab">Organisations</a></li>
-        <% } %>
-
-        <% if(locals.area === 'services') { %>
-        <li class="active">Services</li>
-        <% } else { %>
-        <li><a href="services" role="tab">Services</a></li>
-        <% } %>
-
-        <% if(!locals.isInvitation) { %>
-        <% if(locals.area === 'audit') { %>
-        <li class="active">Audit</li>
-        <% } else { %>
-        <li><a href="audit" role="tab">Audit</a></li>
-        <% } %>
-        <% } %>
-
-    </ul>
-</nav>
```

Comment:
Not sure why this is being replaced when the scope of the ticket is to enforce the wrapping of text. Could you expand on this, please?

Reply:
Paragraph formatting.